app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class OrganisationsController < ApplicationController
```

Comment:
don't this need to be locked down, to admin users only?

Reply:
Yeah, its managed by the backend i.e.

<img width="1552" alt="Screenshot 2019-12-12 at 13 07 28" src="https://user-images.githubusercontent.com/3071606/70714601-5e62b280-1ce0-11ea-8d26-0592eeabf9c7.png">

Reply:
:+1: